### DISCUSSION TITLE
READ ME FIRST: Welcome to tvb-ukbb Discussions!

### DISCUSSION THREAD

**Discussion by noahfl** [McIntosh-Lab/tvb-ukbb#108] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

Enjoy!

**Comment by justinliuyuwang** [comment]
Please be sure to check out our [FAQ](https://github.com/McIntosh-Lab-RRI/tvb-ukbb/wiki/FAQ) for frequently asked questions and commonly encountered mistakes! The page will be regularly updated as our Discussions grow.
